Sources/iOSClientExposurePlayback/Services/ProgramService.swift

```diff
@@ -44,7 +46,7 @@ public class ProgramService {
     ///
     /// Values in milliseconds.
     ///
-    /// - note: A minimum value of `30000` is enforced
+    /// - note: A minimum value of `1200000`is enforced when doing an entitlement request for the next program , before ending the current program.
```

Comment:
Since I don't know Swift or iOS I don't quite understand how this is connected to startValidationTimer now. Is there a 1 second fuzzy factor plus another 1-120s fuzzy factor added by startValidationTimer?
I think this comment may be misleading since the time doesn't seem to be handled here any more (and there's one zero too many in `1200000`) :wink:

Reply:
Yes comment got an extra zero , will fix it . 

No it will not add 1 + 120s , it is either 1 or 120s , but epg validation will enforce it to use 120s